src/pipx/paths.py

```diff
@@ -83,24 +96,40 @@ def home(self) -> Path:
         elif self._fallback_home:
             home = self._fallback_home
         else:
-            home = Path(DEFAULT_PIPX_HOME)
+            home = Path(self._default_home)
         return home.resolve()
 
     @property
     def shared_libs(self) -> Path:
         return Path(self._base_shared_libs or self.home / "shared").resolve()
 
     def make_local(self) -> None:
-        self._base_home = get_expanded_environ("PIPX_HOME")
-        self._base_bin = get_expanded_environ("PIPX_BIN_DIR")
-        self._base_man = get_expanded_environ("PIPX_MAN_DIR")
-        self._home_exists = self._base_home is not None or any(fallback.exists() for fallback in self._fallback_homes)
+        self._base_home = OVERRIDE_PIPX_HOME or get_expanded_environ("PIPX_HOME")
+        self._default_home = DEFAULT_PIPX_HOME
+        self._base_bin = OVERRIDE_PIPX_GLOBAL_BIN_DIR or get_expanded_environ("PIPX_BIN_DIR")
```

Comment:
This might cause the failure? It's hard to spot..

```suggestion
        self._base_bin = OVERRIDE_PIPX_BIN_DIR or get_expanded_environ("PIPX_BIN_DIR")
```

Reply:
Fixed in [11dd6a6](https://github.com/pypa/pipx/pull/1476/commits/11dd6a6395a550d8703516a51b96e7060addd613)